packages/react-core/src/components/Page/examples/PageHorizontalNav.tsx

```diff
@@ -36,9 +36,11 @@ export const PageHorizontalNav: React.FunctionComponent = () => {
 
   return (
     <Page masthead={masthead}>
-      <PageSection>Section 1</PageSection>
-      <PageSection variant="secondary">Section 2 with secondary variant styling</PageSection>
-      <PageSection>Section 3</PageSection>
+      <PageSection aria-label="Section 1">Section 1</PageSection>
```

Comment:
We'll want to just make sure each PageSection on a page has a unique label; adding the example name to the label may help in cases like this ("Horizontal nav example section 1" etc). Other files will need to be checked as well for whether a similar update is needed.